doc/concepts/cable_cell.rst

```diff
@@ -335,16 +335,24 @@ See :ref:`modelgapjunctions`.
 Discretisation and CV policies
 ------------------------------
 
-For the purpose of simulation, cable cells are decomposed into discrete
-subcomponents called *control volumes* (CVs). The CVs are
-uniquely determined by a set of *B* ``mlocation`` boundary points.
-For each non-terminal point *h* in *B*, there is a CV comprising the points
-{*x*: *h* ≤ *x* and ¬∃ *y* ∈ *B* s.t *h* < *y* < *x*}, where < and ≤ refer to the
-geometrical partial order of locations on the morphology. A fork point is
-owned by a CV if and only if all of its corresponding representative locations
-are in the CV.
+.. glossary::
 
-The set of boundary points used by the simulator is determined by a *CV policy*.
+  control volume
+    For the purpose of simulation, Arbor discretises cable cell :term:`morphologies <morphology>`
+    into control volumes, or CVs. Discretising happens through a :term:`CV policy`.
+    The CVs are uniquely determined by a set of *B* :term:`mlocation` boundary points. For each non-terminal
+    point *h* in *B*, there is a CV comprising the points {*x*: *h* ≤ *x* and ¬∃ *y* ∈ *B* s.t *h* < *y* < *x*},
+    where < and ≤ refer to the geometrical partial order of locations on the morphology. A fork
+    point is owned by a CV if and only if all of its corresponding representative locations are
+    in the CV.
+
+.. Note::
+    In NEURON discretisation is controlled through splitting a NEURON section into a number of NEURON segments (`nseg`, by default to `1`). Since NEURON sections don't exactly map to Arbor :term:`segments <segment>`, there is not always a way to match the precise discretisation. Therefore results may be slightly different for otherwise identical simulations.
```

Comment:
I'll say that NRN segment is similar to an Arbor branch.

Can you elaborate on the last paragraph of your comment? Would be nice to include!

Reply:
Neuron's **section** is similar to Arbor's branch.

I'm not very familiar with Neuron's discretization process, but I'm fairly certain we made no effort to accurately reproduce it in Arbor. If I had to guess I would say that a neuron branch (nrn: section) is made of `nseg` equally sized CVs (nrn: segments). Prior to the introduction of CV policies, we made special considerations at fork points, and the CVs at the edges of a branch were half the size of the CVs in the middle of a branch, so the CVs in Arbor and Neuron just didn't match up. This has changed quite a bit with the introduction of CV policies, which have only made the differences more pronounced: we can use several cv policies on the same cell; we can describe exactly how to split a cell into arbitrarily sized CVs; etc (@halfflat can say more about that since he designed and wrote the code!). 

In any case, the discretization not matching exactly between Arbor and Neuron is probably not the biggest source of discrepancy in results. Arbor uses truncated cones to represent branches and segments, Neuron uses their average radius to create cylinders; I've observed that that has more of an effect on the results than the discretization matching up exactly. If the issue is indeed the discretization, then by using a finer discretization in both Arbor and Neuron, both simulators should converge to the same results.

Reply:
Sorry, I mistyped (curse all those similar quite subtly differently defined terms): I meant to write section!

Thanks for the explanation. I've added the cylinder vs cone. @halfflat if you care to elaborate on how Arbor discretises around branching points, please do :)